backend/coreapp/compilers.py

```diff
@@ -123,10 +124,7 @@ def from_id(compiler_id: str) -> Compiler:
 
 @cache
 def available_compilers() -> List[Compiler]:
-    return sorted(
```

Comment:
I think this was sorted intentionally. Is _compilers.values a static order?

Reply:
`_compilers` is an `OrderedDict`, so I assume it's just ordered the way the definition is, that at least reflects what I see locally.